src/components/person.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import './person.scss';
+
+import PropTypes from 'prop-types';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+  let status = '';
+
+  if (sex === 'f') {
+    status = 'husband';
+  } else
+
+  if (isMarried && sex === 'm') {
+    status = 'wife';
+  }
```

Comment:
It's not status, it's some info about partner
```suggestion
  let status = '';

  if (sex === 'f') {
    status = 'husband';
  } else if (sex === 'm') {
    status = 'wife';
  }
```